tests/src/Kernel/CorporateCountriesRdfKernelTestBase.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_corporate_countries\Kernel;
+
+use Drupal\Tests\rdf_entity\Kernel\RdfKernelTestBase;
+use Drupal\Tests\rdf_skos\Traits\SkosImportTrait;
+
+/**
+ * Abstract class for kernel tests with a test RDF country vocabulary.
```

Comment:
"Abstract class for kernel tests with a test using the RDF country vocabulary."

Reply:
"test RDF country vocabulary" or "RDF country test vocabulary". I mean that this test has a test example vocabulary provided.

Reply:
Gotcha